### DISCUSSION TITLE
When will megatron Flash attention 3 be supported？

### DISCUSSION THREAD

**Discussion by echo-valor** [NVIDIA/Megatron-LM#1099] [Q&A]
**Your question**
Flash attention 3 has been updated, link:
- Blogpost: https://tridao.me/blog/2024/flash3/
- Paper: https://tridao.me/publications/flash3/flash3.pdf

Megatron's support for Flash attention 3 to improve training efficiency。FlashAttention-3 is optimized for Hopper GPUs (e.g. H100).

**Comment by IgorZan** [comment]
Hi, any progress about this?
